repos/system_upgrade/el7toel8/actors/multipathconfread/libraries/multipathconfread.py

```diff
@@ -201,3 +208,25 @@ def get_multipath_conf_facts(config_file='/etc/multipath.conf'):
     else:
         res_configs.extend(_parse_config_dir('/etc/multipath/conf.d'))
     return MultipathConfFacts(configs=res_configs)
+
+
+# FIXME: should we use rather list of config files from MultipathConfFacts
```

Comment:
If possible, yes please. Better then HC :)

Reply:
@Rezney  I have done something, but it's possible the change will be reverted yet. Let's sync tomorrow.

Just note for readers:  Waiting for more info about xDR devices. Maybe I will keep the proper fix on separate PR to not create additional complexity here.

Reply:
I have reverted the change and put it into the shape of the original FIXME (more or less). The proper fix will be delivered in separate PR in future.